### PR TITLE
🧪 Various Tests

### PR DESCRIPTION
**Pull request by mrsegen**

### CODE REVIEW

File: src/utils/trading/__tests__/symbol.utils.test.ts [hunk 1]
```diff
@@ -1,9 +1,15 @@
+import { getSpotQuote, getSpotSymbol } from '../symbol.utils';
+
 describe('Symbol utils', () => {
   describe('getSpotSymbol', () => {
```
Comment (mrsegen) [inline src/utils/trading/__tests__/symbol.utils.test.ts:4]: Is it worth writing sad path tests? (e.g. X//Y, X/Y/Z, /Y, X/) This would mean error handling in the utils.
I'm not yet familiar with where this would be coming from to have an opinion whether this makes sense.

File: src/utils/trading/__tests__/conversion.utils.test.ts [hunk 2]
```diff
@@ -1,14 +1,76 @@
...
+      expect(getTokensAmount('symbol', 50, -150)).toEqual(-3);
+      expect(getTokensAmount('symbol', -50, 150)).toEqual(-3);
+      expect(getTokensAmount('symbol', -50, -150)).toEqual(3);
+      expect(getTokensAmount('symbol', 0, 0.01)).toEqual(Infinity);
+      expect(getTokensAmount('symbol', null, -150)).toEqual(-Infinity);
+      expect(getTokensAmount(null, 3, 45)).toEqual(15);
+    });
+
+    it('should throw on error', () => {
+      expect(() => getTokensAmount(null, 2.8, null)).toThrowError(TypeError);
```
Comment (mrsegen) [inline src/utils/trading/__tests__/conversion.utils.test.ts:48]: Let me know what you think about what direction to go in with this type of test

Comment (thibaultyou) [reply src/utils/trading/__tests__/conversion.utils.test.ts:48]: I'm not sure where to go with this, my first intention was to improve existing error handling in order to ensure that we have all we need to do our calculation, you don't really need to test all possible cases, null or NaN should be enough I guess

**Review by thibaultyou — APPROVED** [review]

File: src/utils/trading/__tests__/conversion.utils.test.ts (hunk 2 above)
Comment (mrsegen) [reply src/utils/trading/__tests__/conversion.utils.test.ts:48]: Thanks for the feedback. I'll go lighter moving forward. Feel free to discard anything you like from it
